app.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
We should add all navigation elements to this top level "frame" and render the content inside to DRY Up the code and keep a consistent UI